### DISCUSSION TITLE
Secrets can be inferred from action logs

### DISCUSSION THREAD

**Discussion by pkusunjy** [community/community#119738] [Actions] [answered]
### Select Topic Area

Question

### Body

When I was reviewing action logs, there happened to be one of my secrets variables, replaced with **. If others could see my action logs, they would know I had set a secret variable whose value is 22, according to the context. I am not sure whether there is a security risk, or how can I avoid this case?
![sensitive](https://github.com/community/community/assets/11880269/675a3df1-f1dd-49e2-92a1-3f52396ee90f)

**Comment by BourgoisMickael** [comment, marked as answer]
Why would `22` be a secret ? That's just a variable, it can't protect anything, too easily breakable.
Secrets are for sensitive values that needs to be long and random like passwords or keys.

Any short value that can be brute forced should not be used to restrict access or protect resource and therefore don't need to be a secret I guess.
___

Otherwise you can save it encoded (like in `base64`) and then decode it in your action
